core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -298,9 +298,19 @@ public static void authorizationPluginRenamePrivileges(
           NameIdentifierUtil.toMetadataObject(NameIdentifier.of(ident.namespace(), newName), type);
       MetadataObjectChange renameObject =
           MetadataObjectChange.rename(oldMetadataObject, newMetadataObject);
+
+      String metalake;
```

Comment:
I think you can change to use `? :` operand to simplify your code.

Reply:
Good point. Changed.